playshogi-library-shogi/src/main/java/com/playshogi/library/shogi/models/formats/sfen/SfenConverter.java

```diff
@@ -77,9 +77,7 @@ public static String toSFEN(final ShogiPosition pos) {
             c = "-";
         }
 
-        // Should we add the move count?
```

Comment:
Sorry I should have updated the comment - right now, the SFEN is used as a unique representation of the position.
e.g. in the opening explorer, the client sends the SFEN string to the server to look it up in the database. 2 issues here:
1. changing this representation means that we have to regenerate all database
2. For the game position search, I actually want to ignore the number of moves (once we reach a position, it doesn't matter how many moves it took to get there - i want to show all games in database that have that position).

Could we maybe have a boolean parameter "withMoveCount" to that method, and only use it for the new display panel for now?

Reply:
Ah, right, I forgot that shogi has more transpositions than chess, so it would be best if move count were not considered by game search.  Yes, I can correct this.